### ISSUE TITLE
flandmark_in windows simple_example.m is not running

### ISSUE THREAD

**Issue by arthahir** [uricamic/flandmark#28]
i have the following error when i compile my flandmark_simple_example.m in matlab
how i need use flandmark in windows help me please

Undefined function or variable 'flandmark_load_model'.

Error in flandmark_simple_example (line 17)
model = flandmark_load_model('../data/flandmark_model.dat');

**Comment by uricamic** [comment]
Hi @arthahir,

have you compiled the mex files successfully? There should be .mexw64 or .mexw32 (depending on your system & matlab version) files in /matlab_toolbox/mex/ folder. If not, I can provide you pre-compiled version for windows 7 64bit and 64bit MATLAB: http://www.ulozto.net/x23qdRE/flandmarkv107-precompiled-win7-64b-zip?_ga=1.106888643.1736449076.1428859677

I should also say, that flandmark is now considered to be obsolete, it was replaced by CLandmark: https://cmp.felk.cvut.cz/~uricamic/clandmark/
which is faster, more user friendly and without external dependencies (like OpenCV).

**Comment by arthahir** [comment]
hiiii dude,
                  sure i will have a look at Clandmark.Thank you but the precomplied version ..but when i copied .mexw64 it to matlab_toolbox/mex/.mexw64  and run my simple_example.m code is shows me the following error..

Invalid MEX-file 'D:\flandmark-master\matlab_toolbox\mex\flandmark_load_model.mexw64': The specified module could not be found. 

please help me to run this code..

**Comment by uricamic** [comment]
Hi, 

I am now a bit lost. So, you have downloaded the pre-compiled version and tried to run that and obtained this error?

**Comment by arthahir** [comment]
yes i have downloaded and tried running flandmark_simple_example.m in my matlab and found that error.!

**Comment by uricamic** [comment]
Which version of MATLAB and operating system are you using?

**Comment by arthahir** [comment]
matlab R2015a
windows 7 operating system

is that my operating system may be causing this error!i dont know the exact reason..pls

**Comment by uricamic** [comment]
I guess both is 64bit, right? 

Operating system should not be the problem, since I have compiled it on Windows 7. The version of MATLAB, though, could be a problem. I don't remember correctly, but I guess, I have used MATLAB 2012b...

It seems you will have to compile it by yourself then... Do you use MS Visual Studio?

**Comment by arthahir** [comment]
hii,
yes both are 64 bit...and yes i have MS Visual studio express 2012..how can i go with Visual studio, if matlab is the problem!

**Comment by uricamic** [comment]
MATLAB is not a problem. What I am trying to say is, that the pre-compiled version is not compatible with your system setting (probably just the version of MATLAB is a problem). So, you can still use flandmark in MATLAB, but you will have to compile it by yourself for your system...

If you are using Visual Studio, then I guess, you will have to modify some paths in the project solution file generated by CMake. Unfortunately, I don't have access to a windows machine now. But IIRC, you just need to specify additional paths in linker options. It might be also easier to first compile the library in Visual Studio and then to produce mex-files directly in MATLAB using the enclosed script.

**Comment by arthahir** [comment]
even i have same though matlab could be probelm may be.
in these case let me try this out in visual studio..
